src/core/types/addons.js

```diff
@@ -1,8 +1,12 @@
 /* @flow */
-import type { VersionIdType, VersionLicenseType } from 'core/reducers/versions';
+import type {
+  VersionIdType,
+  ExternalAddonVersionType,
+  PartialExternalAddonVersionType,
+} from 'core/reducers/versions';
 import type { AddonTypeType } from 'core/constants';
 
-type AddonStatus =
+export type AddonStatusType =
```

Comment:
It'd be nice to avoid the `Type` suffix whenever we can. It is more a general comment, but usually we don't need it. I know we are not consistent everywhere, but I don't think we should encourage the "Type" suffix. Flow is all about types, so we already know that we deal with types, and because JS (the language) does not support types anyway, we cannot be confused.

The only exception would be a conflict when importing a type in a file that already has something with the same name, but we could alias the type in this case:

```
import type { Addon as AddonType } from '...';
```

Reply:
I will prepare a list of conventions for the All Hands and we'll discuss each, one by one. You can leave it as is.

Reply:
Ok, that saves me a step, thanks. 😃